pkgs/development/web/rover/default.nix

```diff
@@ -0,0 +1,59 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, perl
+, openssl
+, pkg-config
+, darwin
+, stdenv
+}:
+
+let
+  # The build fetches this dynamically from https://graphql.api.apollographql.com/api/schema
+  apolloSchema = builtins.readFile ./schema.graphql;
+  # The build fetches this from the headers of https://graphql.api.apollographql.com/api/schema
+  etagId = "a7252784c3d195f1ffec315fb8b1481cfba06c7943fe9020ab8df2aafd4d479e";
+  inherit (darwin.apple_sdk.frameworks) Security;
```

Comment:
This should be done in top-level.

Reply:
Not sure what you mean by that comment. Can you elaborate please?

Reply:
https://www.github.com/NixOS/nixpkgs/blob/2e435a34d5bf50cda3883bdb07d6dd03f899b54c/pkgs/top-level/all-packages.nix#L192

Reply:
ah yah makes sense :sweat_smile: done in d27aa5d1754384715f269af0613e957ecfeedcd0